_security/access-control/api.md

```diff
@@ -218,12 +233,23 @@ GET _plugins/_security/api/actiongroups/
     "hidden": false,
     "allowed_actions": [
       "indices:data/read*",
-      "indices:admin/mappings/fields/get*"
+      "indices:admin/mappings/fields/get*",
+      "indices:admin/resolve/index"
     ],
     "type": "index",
     "description": "Allow all read operations",
     "static": true
   },
+  "cluster_all": {
```

Comment:
nicely done.. This should help users understand that there are more than 1 actiongroups in the cluster.